app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2672,11 +2688,16 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
     out.source.on_edge_drop <+ overlapping_edges;
 
     drop_on_bg_up  <- background_up.gate(&connect_drag_mode);
-    drop_edges     <- any (drop_on_bg_up,touch.background.down);
-    edge_to_drop_without_targets <= drop_edges.map(f_!(model.take_edges_with_detached_targets()));
-    edge_to_drop_without_sources <= drop_edges.map(f_!(model.take_edges_with_detached_sources()));
-    edge_to_drop <- any(edge_to_drop_without_targets,edge_to_drop_without_sources);
-    eval edge_to_drop ((id) model.remove_edge(id));
+    drop_edges     <- any (drop_on_bg_up,click_to_drop_edge);
+
+    edge_dropped_to_create_node <= drop_edges.map(f_!(model.edges_with_detached_targets()));
+    out.source.on_edge_drop_to_create_node <+ edge_dropped_to_create_node;
+
+    remove_all_detached_edges <- any (drop_edges, inputs.drop_dragged_edge);
```

Comment:
Is this an event name? I mean, is this a name that means "this happens on this event"? If so, please rename it to a name starting with "on_<description of what happened>" - this is a new naming convention that we are using in FRP, but it was not used yet in all places in this FRP

Reply:
What identifier are you referring to?